astropy/units/equivalencies.py

```diff
@@ -24,22 +26,71 @@
            'pixel_scale', 'plate_scale', 'with_H0']
 
 
+class Equivalency(UserList):
```

Comment:
Why do we subclass from `UserList` rather than from `List` directly? The documentation suggests that now that subclassing from `list` is possible, the case for `UserList` has mostly disappeared [1].

[1] https://docs.python.org/3/library/collections.html#userlist-objects

Reply:
I think it's a little cleaner to subclass from `UserList` personally. We are really just encapsulating a few lists in one object etc.

Reply:
Indeed, we are just encapsulating the main list, but why use `UserList` at all - we do not use any of its features? And it existed only because `list` could not be subclassed before.

Reply:
OK, given this and my comment about `__add__` below, let's stick with `UserList`!